metadata.rb

```diff
@@ -1,7 +1,18 @@
 name 'delivery-base'
-maintainer 'The Authors'
-maintainer_email 'you@example.com'
-license 'all_rights'
-description 'Installs/Configures delivery-base'
-long_description 'Installs/Configures delivery-base'
+maintainer 'Chef Delivery Team'
+maintainer_email 'delivery-team@chef.io'
+license 'Apache 2.0'
+description 'Delivery base cookbook'
+long_description 'Sets up the base things for a node to be use in Chef Delivery'
+
 version '0.1.0'
+
+source_url 'https://github.com/chef-cookbooks/delivery-base'
```

Comment:
For backwards compat if we plan to support anything but the latest Chef we'll need  if respond_to?(:source_url) on the end of each of these